### PR TITLE
Document Lively configuration

### PR DESCRIPTION
**Pull request by synbot**
Clear documentation will be provided in the README describing how to build a Lively API configuration.

### CODE REVIEW

**Comment by baohx2000** [comment]
missing type "enum" with "enumValues" property :-1:

**Comment by nearengine** [comment]
:star:

File: README.md [hunk 1]
```diff
@@ -1,7 +1,166 @@
...
+    ]
+}
+```
+
+- **name**: A `string` containing the display name for the API.
+- **logo**: A `string` containing a valid path to an image to display for the logo
+- **summary**: A `string` containing the summary that will be displayed by default when you click on an API
+- **api**: An `Object` containing connection details for API calls (see below)
+- **oauth2**: An `Object` containing the connection and authentication  information for OAuth2 APIs (see below)
+- **resources**: An array of `Object` containing a unique key and an resource configuration object for each resource. Resources can be nested arbitrarily deep, see the resource configuration section for more details. The recommended structure is to create folders for each top level resource inside your API configuration folder.
```
Comment (paulstatezny) [inline README.md]: > resources: An array of Object containing a unique key and an resource configuration object for each resource. 

Knowing the structure of a resource object, I don't understand what is meant by "containing a unique key." Can you clarify?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,166 @@
...
+    type         : 'string',
+    location     : 'uri',
+    defaultValue : 'synapsestudios'
+    description  : 'The owner of the repo.'
+}
+```
+
+- **name**: A `string` containing the key or name of this parameter that will be sent in the request body or injected into the URI.
+- **required**: A `bool` (`true` or `false`) which flags this parameter as required.
+- **type**: A `string` containing one of: `string`, `integer`, `boolean`, `resumable-upload`, `hash`, `array[hash]`, `array[string]`, or `enum`. If `enum`, specify `enumValues` property on param containing an array of strings to display as options to the user.
```
Comment (paulstatezny) [inline README.md]: `array[boolean]` and `array[integer]` can be added to this list.

**Comment by paulstatezny** [comment]
This is great documentation; thanks for writing it. Left a couple comments. :interrobang:

**Comment by nearengine** [comment]
:star:

**Comment by areida** [comment]
Looks good to me.

**Comment by zpchavez** [comment]
Looks good. Will need to add 'file' to accepted param types when #106 is merged.

**Comment by paulstatezny** [comment]
:+1: We can make an issue for `file` after #106 is finished.

**Pull request merged**